str.d.ts

```diff
@@ -0,0 +1,21 @@
+import type {Props} from './dom.d.ts';
+
+export {Raw, boolAttrs, voidElems, cls, len, /* TODO: map */} from './dom.d.ts';
+
+type Child = string | Child[];
+
+export function E(name: string, props: Props, ...children: Child[]): string;
```

Comment:
This returns `Raw` (subclass of `String`) rather than `string`. The distinction is important. Prax doesn't escape `String` instances, but _does_ escape `string` values. An instance of `String` cannot be passed to functions that require a plain `string`, and must be converted via `.toString()`. I think it's important to check this at the type level.

This also applies to `S`, `F`, `e`.